docs/section19/19-13.md

```diff
@@ -0,0 +1,75 @@
+                - Check if the zone_design_thermostat_heating_setpoint is greater than the design_thermostat_heating_setpoint: `if zone.zone_design_thermostat_heating_setpoint > design_thermostat_heating_setpoint:`  
+                    - Set the design setpoint equal to the zone design setpoint: `design_thermostat_heating_setpoint = zone.zone_design_thermostat_heating`  
+            
+            Below checks for a delta T of 20F:
+            - For each zone in the list of zones served by the hvac system: `for zone in zones_list_hvac_sys_b:`  
+                - Reset zone has lab space boolean variable: `zone_has_lab_space = false`  
+                - For each space in zone: `for space in zone.spaces:`
+                    - Check if space is a lab based on lighting_space_type: `if space.lighting_space_type in ["LABORATORY_EXCEPT_IN_OR_AS_A_CLASSROOM"]: zone_has_lab_space = true`
+                - For each terminal unit associated with the zone: `for terminal in zone.terminals:`  
+                    - Check if the terminal unit is associated with the hvac system: `if terminal in terminal_list_hvac_sys_b:`  
+                        - Get the design heating supply temperature setpoint: `design_heating_supply_air_temp_setpoint = terminal.supply_design_heating_setpoint_temperature` 
+                        - Get the design cooling supply temperature setpoint: `design_cooling_supply_air_temp_setpoint = terminal.supply_design_cooling_setpoint_temperature` 
+                        - Check if the zone serves at least one lab space: `if zone_has_lab_space == true:`  
+                            - Check if the delta temperature is NOT 17F for heating: `if design_heating_supply_air_temp_setpoint-zone_design_thermostat_heating_setpoint != 17:all_design_setpoints_delta_Ts_are_per_reqs = false`  
+                            - Check if the delta temperature is NOT 17F for cooling: `if zone_design_thermostat_cooling_setpoint - design_cooling_supply_air_temp_setpoin != 17:all_design_setpoints_delta_Ts_are_per_reqs = false`  
+                        - Else, zone does not serve a lab space: `Else:`  
+                            - Check if the delta temperature is NOT 20F for heating: `if design_heating_supply_air_temp_setpoint-zone_design_thermostat_heating_setpoint != 20:all_design_setpoints_delta_Ts_are_per_reqs = false`  
+                            - Check if the delta temperature is NOT 20F for cooling: `if zone_design_thermostat_cooling_setpoint - design_cooling_supply_air_temp_setpoin != 20:all_design_setpoints_delta_Ts_are_per_reqs = false`  
+
+        - **Rule Assertion:** 
+        - Case 1: If the supply-air-to-room temperature set-point difference equals 20°F or 17F per G3.1.2.8.1 including Exceptions 1, the fan is autosized and the FanSystem.minimum_airflow>FanSystem.Minimum_Outdoor_Airflow: `if all_design_setpoints_delta_Ts_are_per_reqs == true and are_all_hvac_sys_fan_objs_autosized(B_RMI, hvac) == True : outcome = "PASS"`  
```

Comment:
For labs, we are not checking the part that "or the required ventilation air or makeup air, whichever is greater". We have FanSystem.minimum_outdoor_airflow attribute that can potentially be used for this.

Reply:
Assume what they specified in the proposed design is the ventilation required. if does not meet 20 or 17 (if lab) then see if the unit is modeled as min OA = supply CFM based on proposed OA assuming to meet code requirements.

Reply:
This does not apply to just labs. Should we include this for all units?

Reply:
Updated per discuss for 19-17 which is analogous.